lisa/testsuite.py

```diff
@@ -94,8 +94,7 @@ def __post_init__(self, *args: Any, **kwargs: Any) -> None:
         self._timer: Timer
 
         self._environment_information: Dict[str, Any] = {}
-        # parent_log = get_logger("suite", self.runtime_data.metadata.suite.name)
-        self.log = get_logger("case", self.name)
+        self.log = get_logger("case", f"{self.name}[{self.id_}]")
 
```

Comment:
@squirrelsc  @LiliDeng 
Sample log

2025-07-30 05:10:06.840[7020][DEBUG] lisa.case[smoke_test[lisa_1_0]] case end in 27.463 sec
2025-07-30 05:10:06.841[7020][DEBUG] lisa.case[smoke_test[lisa_1_0]] after_case end in 0.000 sec
2025-07-30 05:10:06.841[7020][INFO] lisa.case[smoke_test[lisa_1_0]] result: PASSED, elapsed: 44.042 sec

Reply:
Use `()` like `self.log = get_logger("case", f"{self.name}({self.id_})")`

Reply:
Agreed, updated

2025-07-31 04:45:42.993[136335869052480][DEBUG] lisa.case[smoke_test][lisa_0_0] case end in 248.536 sec
2025-07-31 04:45:42.994[136335869052480][DEBUG] lisa.case[smoke_test][lisa_0_0] after_case end in 0.001 sec
2025-07-31 04:45:42.994[136335869052480][INFO] lisa.case[smoke_test][lisa_0_0] result: PASSED, elapsed: 251.734 sec